views/js/controller/runner/runner.js

```diff
@@ -23,6 +23,7 @@
  */
 define([
     'jquery',
+    'select2', // TODO: remove as soon as PCI stopped expose jQuery on window
```

Comment:
Since the module hooks on jQuery and does not return anything, better to put it at the end of the list. This way you won't need to reserve a variable name (which will have an undefined value BTW).

Reply:
Unfortunately we have to start loading plugin as soon as possible.
Anyway we will remove it next sprint